### PR TITLE
Search results improvements

### PR DESCRIPTION
**Pull request by smth**
Addresses most of #305

### CODE REVIEW

File: assets/js/vendor/index.js [hunk 1]
```diff
@@ -3,3 +3,12 @@
 require('nodep-date-input-polyfill');
 
 require('jquery.mmenu');
+
+require('tooltipster');
+
+$(document).ready(function() {
+  $('.hastip').tooltipster({
+     trigger: 'click',
```
Comment (vitorbaptista) [inline assets/js/vendor/index.js]: Can we enable hover as well?

Comment (vitorbaptista) [inline assets/js/vendor/index.js]: Also, move it to https://github.com/opentrials/opentrials/blob/master/assets/js/index.js instead. This file should only contain the `requires`.

**Comment by vitorbaptista** [comment]
I also found a pretty minor bug on the advanced search. To reproduce, do:

1. Open the search page with a big resolution (to get the "desktop version")
2. Reduce the screen size until you get the mobile version.
3. Click on `Advanced search` to expand the options, then click again to remove them
4. Increase the screen size to get back to the desktop version

The advanced search filters will be hidden, and the `Advanced search` button won't be clickable (so you can't expand them).

If that's difficult to fix, I'm happy to just create an issue.

advanced search bug

**Comment by vitorbaptista** [comment]
Also, please check the Travis build. There's a minor error with linting the JS code.

File: assets/js/vendor/index.js (hunk 1 above)
Comment (smth) [inline assets/js/vendor/index.js]: We can have hover for mouse and tap for touch, if that's what you're alluding to. Having hover and click for mouse seems unnecessary.

**Comment by smth** [comment]
@vitorbaptista What do you mean by odd in regards to the filters?

I was aware of that issue with resizing, but it seems like a pretty unlikely scenario to me.

**Pull request merged**